### DISCUSSION TITLE
📝 Fill out a quick survey

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1903] [Announcements]
We’ve just released **TE Stable Version** - check out [the Changelog](https://tailwind-elements.com/docs/standard/getting-started/changelog/#v1.0.0).

Right now we want to **gather as much feedback as we can** on the Stable Version & keep improving it.

Please answer some basic questions regarding your experience with Tailwind Elements.

**[📝 FILL OUT A QUICK SURVEY](https://forms.gle/ux6UeYT41XEEDaDM7)**

**Comment by nikkimarie31** [comment]
im trying to follow your tutorial and im on the mdb go section and when i download the mdb-cli and did the command that created the project all was good up until i typed npm start then it said this 

npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
checksum failure
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_ENTRY_INVALID 
invalid base256 encoding       
npm WARN tar TAR_BAD_ARCHIVE: Unrecognized archive format     
npm ERR! code TAR_BAD_ARCHIVE
npm ERR! TAR_BAD_ARCHIVE: Unrecognized archive format

im not sure how to proceed. i already tried restarting from a different location and then tried on my bash terminal then on my vs code terminal. Still getting the exact same errors. i tried contacting you on twitter and it wants me to pay to send you a message 

thank you

**Reply by filipkappa** [reply]
It seems like you have some invalid installs.

Please try to [refresh your cache first](https://coder-coder.com/npm-clear-cache/), and if that doesn't work try **[re-installing node](https://www.mend.io/free-developer-tools/blog/reinstall-npm-node-js/)**.
